.github/labeler.yml

```diff
@@ -0,0 +1,39 @@
+algorithms:
+  - src/pymortests/algorithms/*
+  - src/pymor/algorithms/*
+
+buildsystem:
+  - .ci/*
+  - .github/*
+  - .binder/*
+  - Makefile
+  - azure-pipelines.yml
+  - requirements*
+  - pyproject.toml
+  - MANIFEST.in
+  - dependencies.py
+  - setup.*
+  - versioneer.py
+
+bindings:
+  - src/pymor/bindings/*
+
+builtin-models:
+  - src/pymor/discretizers/*
+  - src/pymor/playground/discretizers/*
+  - src/pymortests/model.py
+  - src/pymortests/fixtures/model.py
+  - src/pymor/models/*
```

Comment:
The code here is completely generic. 'builtin-models' only refers to `Models` that are created using pyMOR's discretization toolkit or at at least are based on a pyMOR `analyticalproblem`. (Currently both classes are the same.)